examples/providers/opentok/src/main.ts

```diff
@@ -0,0 +1,87 @@
+import { Source } from "./source";
+import { Target } from "./target";
+
+import { OpenTokFacade } from "./facades/opentok_facade";
+import { Pipeline } from "./pipeline";
+import { FaceColorTransformer } from "./transformers/face_color_transformer";
+import {
+    VolumeOwnerTransformer,
+    VolumeExtractorTransformer,
+} from "./transformers/volume_extractor_transformer";
+import { VolumeHistogramTransformer } from "./transformers/volume_histogram_transformer";
+import { vec3 } from "./types";
+import { bindButtonToLink, bindSwitch, bindSlider } from "./utils/ui";
+
+const DEFAULT_MIN_VOLUME: number = 0.0001;
+const DEFAULT_MAX_VOLUME: number = 0.01;
+const DEFAULT_COLOR: vec3 = [255, 0, 0];
+const API_KEY = "47568411";
```

Comment:
lets not put real keys here.. just fake one.

Reply:
But the app won't work then

Reply:
in the readme file we tell them to replace it...
this key can be used for free video calls LOL

Reply:
and anyway this token is expired so we are safe... but lets not push tokens (mostly not to public repos)